SPECS/cairo/cairo.spec

```diff
@@ -1,46 +1,57 @@
 Summary:        A 2D graphics library.
 Name:           cairo
-Version:        1.17.4
-Release:        3%{?dist}
+Version:        1.18.0
+Release:        1%{?dist}
 License:        LGPLv2 OR MPLv1.1
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Libraries
 URL:            https://cairographics.org
-Source0:        https://cairographics.org/snapshots/%{name}-%{version}.tar.xz
+#URL:           https://gitlab.freedesktop.org/cairo/cairo
```

Comment:
Leftover? Or do you want to show there's also an alternative place to read more about this project? One URL is enough but if it's the latter case, please only add a short comment to explain why there's a commented-out `URL` tag.

Reply:
It was the latter, adding a small comment